modules/core/src/test/scala/com.snowplowanalytics.snowplow.bigquery/processing/ProcessingSpec.scala

```diff
@@ -189,10 +198,15 @@ class ProcessingSpec extends Specification with CatsEffect {
     else TestControl.executeEmbed(io)
   }
 
-  def alter1        = alter1_base(legacyColumns = false, timeout = false)
-  def alter1_legacy = alter1_base(legacyColumns = true, timeout = true)
+  def alter1             = alter1_base(legacyColumns = false, timeout = false, legacyColumnMode = false)
+  def alter1_legacy      = alter1_base(legacyColumns = true, timeout = true, legacyColumnMode = false)
+  def alter1_full_legacy = alter1_base(legacyColumns = true, timeout = true, legacyColumnMode = true)
```

Comment:
What is the difference between `alter1_legacy` and `alter1_full_legacy`?  It looks like you are testing exactly the same thing twice.

Reply:
Yeah, it doesn't have a meaning as it is. I'll remove it.